tutorials/i18n/internationalizing_games.rst

```diff
@@ -115,6 +122,18 @@ allow translators to choose the *order* in which placeholders appear:
     # Additionally, this form gives more context for translators to work with.
     message.text = tr("{character} picked up the {weapon}").format({character = "Ogre", weapon = "Sword"})
 
+ .. code-tab:: csharp
+
+    // The placeholder's locations can be changed, but not their order.
+    // This will probably not suffice for some target languages.
+    GetNode<Label>("message").Text = string.Format(Tr("{0} picked up the {1}"), "Ogre", "Sword");
+
+    // The placeholder's locations and order can be changed.
+    // Additionally, this form gives more context for translators to work with.
+    string character = "Ogre";
+    string weapon = "Sword";
+    GetNode<Label>("message").Text = Tr($"{character} picked up the {weapon}");
```

Comment:
I was a bit stumped with how to most accurately solve this, I had thought about using `.replace` after the `Tr` function as well but still not 100% sure what would get the point across like the  GDScript example does

Reply:
Something like this may be better @raulsntos ? 

```csharp
GetNode<Label>("message").Text = Tr("{character} picked up the {weapon}")
            .Replace("{character}", "Ogre")
            .Replace("{weapon}", "Sword");
```

Reply:
Although replace would be inefficient if you had to chain a whole bunch due to creating a new string each time

Reply:
Yeah, I think it may be better to exclude this example from the C# code.